python/tvm/auto_scheduler/search_task.py

```diff
@@ -178,6 +178,12 @@ class SearchTask(Object):
         The target host device of this search task.
     hardware_params : Optional[HardwareParams]
         Hardware parameters used in this search task.
+    layout_rewrite_option : LayoutRewriteOption = LayoutRewriteOption.NO_REWRITE
+        The default layout rewrite option used during program measuring.
+        Cost model will adjust the auto scheduler to find a better schedule for the specified
+        layout rewrite option.
```

Comment:
I think we can directly say "auto_scheduler will find a better schedule for the specified layout rewrite option". Cost model guided search is too detail for end users.